docs/services/collaboration-architect.md

```diff
@@ -0,0 +1,65 @@
+# Collaboration Architect
+
+Collaboration is at the heart of what makes our hackweeks so enlivening and successful. The role of the Collaboration Architect is to identify and remove barriers that inhibit the participants' ability to voice and contribute to their hackweek experience.  The Collaboration Architect develops strategies, processes, and structured conversations that amplify inclusion, shared understanding, brainstorming, co-mentoring, and collaboration.
+
+::::{grid}
+:gutter: 2
+
+:::{grid-item}
+:outline:
+![collaborate](../images/collaboration-collage1.png)
+:::
+:::{grid-item}
+:outline:
+![collaborate](../images/collaboration-collage2.png)
+:::
+:::{grid-item}
+:outline:
+![collaborate](../images/collaboration-collage3.png)
+:::
+
+::::
+
+## Goals
+
+* Reduce the overwhelm and social anxiety that participants commonly experience as they learn multiple topics and are asked to practice using them within project teams, in a tightly compressed week
+* Support early career participants as they step outside their comfort zone to propose and pitch project ideas and step into leading a project team
+* Foster inclusive and generative group work in project teams
+* Guide the co-development of a cohesive portfolio of tutorials by the Education Team
+* Develop opportunities to create longer-lasting connections with each other
+* Cultivate a supportive community that celebrates everyone's accomplishments together
+
+## Range of Activities
```

Comment:
Suggest: Range of Hackweek Activites

Then remove 'Hackweek' from below sub-headings.